packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_GenericDistribution.res

```diff
@@ -262,6 +262,8 @@ let dispatchToGenericOutput = (
     Helpers.toDistFn(ToSampleSet(Belt.Int.fromFloat(float)), dist, ~env)
   | ("toSampleSet", [EvDistribution(dist)]) =>
     Helpers.toDistFn(ToSampleSet(env.sampleCount), dist, ~env)
+  | ("toInternalSampleArray", [EvDistribution(SampleSet(dist))]) =>
```

Comment:
why `Internal`?

Reply:
I wanted to make it clear that this was very distinct from sampleN. That function will randomly sample a distribution, this one will directly take the samples, preserving order.